docs/content/guides/author-apps/dapr/how-to-dapr-building-block/snippets/app-statestore.bicep

```diff
@@ -0,0 +1,46 @@
+import radius as radius
+
+@description('The ID of your Radius Application. Automatically injected by the rad CLI.')
+param application string
+
+@description('The ID of your Radius environment. Set automatically by the rad CLI.')
+param environment string
+
+//CONTAINER
+resource demo 'Applications.Core/containers@2023-10-01-preview' = {
+  name: 'demo'
+  properties: {
+    application: application
+    container: {
+      image: 'ghcr.io/radius-project/samples/demo:latest'
+      ports: {
+        web: {
+          containerPort: 3000
+        }
+      }
+    }
+    extensions: [
+      {
+        kind: 'daprSidecar'
+        appId: 'demo'
+        appPort: 3000
+      }
+    ]
+     connections: {
```

Comment:
The whitespace doesn't seem to line up here. Could you double check this please?